### PR TITLE
Default a PMC command with no -Source parameter to the currently selected value

### PR DESCRIPTION
**Pull request by joelverhagen**
Fixes https://github.com/NuGet/Home/issues/3211. See issue for more details.

/cc @mishra14 @emgarten @alpaix

### CODE REVIEW

File: src/NuGet.Clients/PackageManagement.PowerShellCmdlets/NuGetPowerShellBaseCommand.cs [hunk 1]
```diff
@@ -326,15 +326,18 @@ protected string CheckSourceValidity(string source)
         /// <param name="matchingSource">Matching Source if the specified source is a known source. Else null.</param>
         protected void UpdateActiveSourceRepository(string source, bool validateSource)
         {
+            // If source string is not specified, get the current active package source from the host
+            source = string.IsNullOrEmpty(source) ? (string)GetPropertyValueFromHost(ActivePackageSourceKey) : source;
```
Comment (rohit21agrawal) [inline src/NuGet.Clients/PackageManagement.PowerShellCmdlets/NuGetPowerShellBaseCommand.cs:330]: would it make sense to move it to CheckSourceValidity if the source is empty or null?

**Comment by rohit21agrawal** [comment]
@mishra14 would it be possible to include a test case for this?

**Comment by rohit21agrawal** [comment]
@joelverhagen just curious, how does this work when Package Source is selected as All?

**Comment by joelverhagen** [comment]
The `.Value` of the dropdown "All" item is an empty string. So all the validation stuff essentially no-ops.

File: src/NuGet.Clients/PackageManagement.PowerShellCmdlets/NuGetPowerShellBaseCommand.cs (hunk 1 above)
Comment (mishra14) [inline src/NuGet.Clients/PackageManagement.PowerShellCmdlets/NuGetPowerShellBaseCommand.cs:330]: CheckSourceValidity is called only when validateSource bool is True. When cmdlets call UpdateActiveSourceRepository then the bool is False. Not sure if we can move it inside CheckSourceValidity.

TL;DR - CheckSourceValidity is not called on all calls to UpdateActiveSourceRepository

**Comment by mishra14** [comment]
@rohit21agrawal  : looking into it.

**Comment by joelverhagen** [comment]
I don't believe we have any PMC Xunit tests yet. It might be a significant amount of test work to allow this. Still a good idea though, but should not block this PR.

**Comment by rohit21agrawal** [comment]
LGTM :shipit:

**Comment by joelverhagen** [comment]
Tracking PMC Xunit tests: https://github.com/NuGet/Home/issues/3213

**Comment by mishra14** [comment]
:shipit:
